kubetest2/kubetest2-eks/main.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/aws/aws-k8s-tester/eks"
```

Comment:
minor nit: looks like this is actually the deployer implementation.
this might get confusing later since "tester" has a different meaning in the kubetest2 sense.

might want to rename this later (just something to keep at the back of our minds) .